### ISSUE TITLE
Implement auto-update

### ISSUE THREAD

**Issue by fmaclen** [fmaclen/canutin#28]
https://www.electron.build/auto-update

Blocked by https://github.com/Canutin/desktop-2/issues/29

**Comment by fmaclen** [comment]
Superseded. v2 has moved away from the v1 Electron architecture. Auto-update for the new Electron wrapper will be tracked in a separate issue.
